### ISSUE TITLE
Fetch not working

### ISSUE THREAD

**Issue by ZotyDev** [juraj-hrivnak/Pakku#14]
After 0.11.1, when I try to `pakku fetch` it throws this error:

```
Exception in thread "main" java.nio.file.AccessDeniedException: .\mods\1.12.2
        at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:89)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
        at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
        at java.base/java.nio.file.Files.newByteChannel(Files.java:380)
        at java.base/java.nio.file.Files.newByteChannel(Files.java:432)
        at java.base/java.nio.file.Files.readAllBytes(Files.java:3288)
        at teksturepako.pakku.cli.cmd.Fetch$run$1.invokeSuspend(Fetch.kt:94)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.internal.ScopeCoroutine.afterResume(Scopes.kt:32)
        at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at teksturepako.pakku.cli.cmd.Fetch.run(Fetch.kt:27)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:306)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:319)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:40)
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:458)
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:455)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:475)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:482)
        at teksturepako.pakku.MainKt.main(Main.kt:24)
```

**Comment by juraj-hrivnak** [comment]
Remove the 1.12.2 folder from the mods folder
I will fix this later

**Comment by juraj-hrivnak** [comment]
Fixed in v0.11.2
